utils/src/main/kotlin/Utils.kt

```diff
@@ -381,3 +381,10 @@ fun String.fileSystemEncode() =
 fun String.fileSystemDecode(): String =
         // URLDecoder does decode "." and "*".
         java.net.URLDecoder.decode(this, "UTF-8")
+
+/**
+ * Print the stack trace of the [Throwable] if [printStackTrace] is set to true.
+ */
+fun Throwable.logStackTrace() {
```

Comment:
I'm not so happy about the "log" in the name as it's not really using the logger. To avoid another use of "print", maybe "showStackTrace"?

Reply:
Done.